src/util/octokit.ts

```diff
@@ -18,9 +18,19 @@ export const paginate = (
   while (octokit.hasNextPage(response)) {
     // Avoid GitHub's "abuse detection mechanisms"
     await wait(2000);
-
-    response = await octokit.getNextPage(response); // eslint-disable-line no-await-in-loop
-    data = data.concat(extractItems(response.data));
+    try {
+      response = await octokit.getNextPage(response); // eslint-disable-line no-await-in-loop
+      data = data.concat(extractItems(response.data));
+    } catch (e) {
+      if (e.headers && e.headers['retry-after']) {
+        const retryAfter = e.headers['retry-after'];
```

Comment:
We should parse this to a number and handle invalid values. I suggest `Number(e.headers['retry-after']))` to parse the number and throwing if `Number.isNaN(retryAfter)` to handle invalid values.